doc/source/language/flow.rst

```diff
@@ -384,105 +386,324 @@ Halts execution for a specified amount of time, or until a specific set of crite
     WAIT UNTIL X > 40.            // Wait until X is greater than 40
     WAIT UNTIL APOAPSIS > 150000. // You can see where this is going
 
-Note that any ``WAIT`` statement, no matter what the actual expression is, will always result in a wait time that lasts at least :ref:`one physics tick <cpu hardware>`.
+Note that any ``WAIT`` statement, no matter what the actual expression is, will always result in a wait time that lasts at least :ref:`one physics tick <physics tick>`.
 
-.. note::
+Difference between wait in mainline code and trigger code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When called from your mainline code, the :ref:`WAIT <wait>`
+command causes mainline code to be suspended, but does
+not stop :ref:`triggers <triggers>` from interrupting
+this waiting period.  Triggers will continue to fire off
+during the time that mainline code is stuck on a wait.
+
+But when a ``WAIT`` is used in a trigger's body
+(A "trigger" is any ``WHEN``, or ``ON`` statement,
+or the expression in a steering control lock like
+``lock throttle to mythrottlefunction().``), it actually
+causes all execution including other triggers to get 
+stuck until the wait is done.  Because of this, while
+it is allowed, it is 
+:ref:`usually a bad idea to use WAIT inside a trigger <wait_in_trigger>`.
 
-    The :ref:`WAIT <wait>` command only causes mainline code
-    to be suspended.  Trigger code such as WHEN, ON, LOCK STEERING,
-    and LOCK THROTTLE, will continue executing while your program
-    is sitting still on the WAIT command.  Furthermore, you cannot
-    cause a WAIT to happen when executing a trigger's body like
-    WHEN or ON, or in the formula for a LOCK STEERING or LOCK
-    THROTTLE or LOCK WHEELSTEERING or LOCK WHEELTHROTTLE statement.
     
+.. index:: Boolean Operators
+.. _booleans:
 
-.. index:: WHEN
-.. _when:
+Boolean Operators
+-----------------
 
-``WHEN`` / ``THEN``
--------------------
+All conditional statements, like ``IF``, can make use of boolean operators. The order of operations is as follows:
 
-Executes a command when a certain criteria are met. Unlike ``WAIT``, ``WHEN``
-does not halt execution. It starts a check in the background that will keep actively looking for the trigger condition while the rest of the code continues. When it triggers, the body after the ``THEN`` will execute exactly once, after which the trigger is removed unless the ``PRESERVE`` is used, in which case the trigger is not removed.
+- ``=`` ``<`` ``>`` ``<=`` ``>=`` ``<>``
+- ``AND``
+- ``OR``
+- ``NOT``
```

Comment:
I did not edit this section.  This is git's diffing getting massively confused by the fact that I re-ordered things and moved the when/then section to the bottom of the page.  It therefore claims, incorrectly, that I added these lines.  What I did was delete the section that came before them, and put it at the end and edit it a lot, which effectively moves these lines up as far as git can tell.  Which it shows as "added in new location, deleted in old location".

So while you're right, the error was in there already.  We can fix it here if you like, but it's outside the remit of this PR.